toolbox/src/main/java/com/asf/appcoins/toolbox/MainActivity.java

```diff
@@ -24,6 +24,7 @@
 public class MainActivity extends Activity {
 
   private static final String TAG = MainActivity.class.getSimpleName();
+  private static final String TAG2 = "THIS_IS_A_GREAT_TEST";
```

Comment:
Maybe change name of the tag to understand to what is related to

Reply:
it's not used, deleted